src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.epics.js

```diff
@@ -11,56 +12,97 @@ import {
     showLoadingViewOnEnrollmentPage,
     successfulFetchingEnrollmentPageInformationFromUrl,
     openEnrollmentPage,
+    startFetchingTeiFromEnrollmentId,
+    startFetchingTeiFromTeiId,
 } from './EnrollmentPage.actions';
 import { urlArguments } from '../../../utils/url';
 
 const fetchEnrollment = id => getApi().get(`enrollments/${id}`, { fields: 'trackedEntityInstance,program,orgUnit' });
 const fetchTrackedEntityInstance = id => getApi().get(`trackedEntityInstances/${id}`, { fields: 'attributes,enrollments' });
 
+const sortByDate = (enrollments = []) => enrollments.sort((a, b) =>
+    moment.utc(b.enrollmentDate).diff(moment.utc(a.enrollmentDate)));
+const deriveSelectedName = (attributes = {}) => attributes.reduce((acc, { value: dataElementValue }) =>
+    (acc ? `${acc} ${dataElementValue}` : dataElementValue), '');
```

Comment:
Still need to sort this out, and filter only the first two as we discussed. I will do together with the https://jira.dhis2.org/browse/DHIS2-10370